src/test/java/pl/psnc/dei/service/TranscriptionPlatformServiceTest.java

```diff
@@ -36,151 +38,155 @@
     public void test() {
```

Comment:
It is not needed, it was added when the test was commented out. Please remove it.